vyper/semantics/types/utils.py

```diff
@@ -181,7 +180,7 @@ def get_index_value(node: vy_ast.Index) -> int:
     # TODO: revisit this!
     from vyper.semantics.analysis.utils import get_possible_types_from_node
 
-    value = node.get("value")
+    value = node
```

Comment:
i guess we don't really need this variable rename anymore -- let's get rid of the `value` variable